### PR TITLE
Reword mailer specs doc strings for ensuring deliveries are cleared

### PR DESCRIPTION
**Pull request by JonRowe**

### CODE REVIEW

**Comment by benoittgt** [comment]
Thanks. Much better.

File: spec/rspec/rails/configuration_spec.rb [hunk 1]
```diff
@@ -257,8 +257,8 @@ def self.application; end
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
 
-    describe 'leans test mailbox after each example' do
-      let(:base_mailer) do
+    describe 'clears ActionMailer::Base::Deliveries after each example' do
```
Comment (pirj) [inline spec/rspec/rails/configuration_spec.rb]: Sorry for the late appearance. Do you think it makes sense to add `order: :defined` here?

Comment (benoittgt) [reply spec/rspec/rails/configuration_spec.rb]: Their is no need. You need at least two example, order does not mater. :)

Comment (JonRowe) [reply spec/rspec/rails/configuration_spec.rb]: Yep they are order independant, they just need different emails.

Comment (pirj) [reply spec/rspec/rails/configuration_spec.rb]: 👍

**Pull request merged**